src/Symfony/Bundle/SecurityBundle/Resources/config/security_authenticator.php

```diff
@@ -143,6 +143,7 @@
                 abstract_arg('options'),
                 service('property_accessor')->nullOnInvalid(),
             ])
+            ->call('setTranslator', [service('translator')->nullOnInvalid()])
```

Comment:
I think `ignoreOnInvalid()` is better (it results in the setter not being called if the `translator` service is not available)

same below

Reply:
Good idea :)